navigator/backend/src/test/scala/com/digitalasset/navigator/backend/ServerTest.scala

```diff
@@ -99,45 +109,18 @@ class ServerTest
   }
 
   it should "forbid to SignIn with a non existing user" in withCleanSessions {
```

Comment:
At some point, we should probably also have a `it should "forbid to SignIn while still connecting to a ledger"` and `it should "forbid to SignIn when permanently failed to connect"`

Reply:
Some point is https://github.com/digital-asset/daml/pull/3138/commits/588e11b6524b1b6e66d5a6bd82dd5e1729484974 :slightly_smiling_face: 

Very good point, thanks for bringing this up.